cli/operator/config/operator-config-example.yaml

```diff
@@ -3,10 +3,19 @@
 # To learn more about operator fields
 # https://github.com/Layr-Labs/eigenlayer-contracts/blob/92ccacc868785350973afc15e90a18dcd39fbc0b/src/contracts/interfaces/IDelegationManager.sol#L21:
 operator:
+  # This is the standard Ethereum address format (ex: 0x6a8c0D554a694899041E52a91B4EC3Ff23d8aBD5) of your operator
+  # which is the ecdsa key you created or imported using EigenLayer CLI
   address:
-  earnings_receiver_address: 
+  # This is the standard Ethereum address format (ex: 0x6a8c0D554a694899041E52a91B4EC3Ff23d8aBD5)
+  # This is the address where your operator will receive earnings. This could be same as operator address
+  earnings_receiver_address:
+  # This is the standard Ethereum address format (0x...)
+  # This is the address which will approve delegation requests.
```

Comment:
expand a bit more: `...address which operator will use to approve delegation requests from stakers`? (If this is what it means)